src/netspresso_trainer/models/op/custom.py

```diff
@@ -172,6 +172,19 @@ def forward(self, x: Union[Tensor, Proxy]) -> Union[Tensor, Proxy]:
         return x
 
 
+class Pool(nn.Module):
+    def __init__(self,
+                 method: str = "max",
+                 kernel_size: int = 2,
+                 **kwargs):
+        super().__init__()
```

Comment:
`MaxPool2d` and `AvgPool2d` have `kernel_size`, `stride`, `padding` parameters as common.
To clarity, I think `stride`, `padding` should also added as parameters.

- https://pytorch.org/docs/stable/generated/torch.nn.MaxPool2d.html
- https://pytorch.org/docs/stable/generated/torch.nn.AvgPool2d.html